tools/ci_build/build.py

```diff
@@ -16,191 +14,324 @@
 import sys
 import hashlib
 import itertools
-from os.path import expanduser
 
-logging.basicConfig(format="%(asctime)s %(name)s [%(levelname)s] - %(message)s", level=logging.DEBUG)
+
+logging.basicConfig(
+    format="%(asctime)s %(name)s [%(levelname)s] - %(message)s",
+    level=logging.DEBUG)
 log = logging.getLogger("Build")
 
+
 class BaseError(Exception):
     """Base class for errors originating from build.py."""
     pass
 
+
 class BuildError(BaseError):
     """Error from running build steps."""
     def __init__(self, *messages):
         super().__init__("\n".join(messages))
 
+
 class UsageError(BaseError):
     """Usage related error."""
     def __init__(self, message):
         super().__init__(message)
 
-def checkPythonVersion():
+
+def _check_python_version():
     # According to the BUILD.md, python 3.5+ is required:
-    # Python 2 is definitely not supported and it should be safer to consider it wont run with python 4:
```

Comment:
be careful with the format/alignment, it seems almost all the lines in the file are changed, it's hard to find your changes.

Reply:
yes, better not to completely modify build.py in this PR 
if we need to clean up something, use a separate PR.

Reply:
Removed ACL changes from this PR, I'll create a new PR for this.